### PR TITLE
[bitnami/airflow] Add support to setup db on k8s job

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR adds support for setting up the database on a k8s job instead of relying on the Webserver setup logic.

### Benefits

- Decoupling init logic from Webserver setup.
- Improve bootstrapping.

### Possible drawbacks

None

### Applicable issues

None

### Additional information

N/A

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by carrodher — DISMISSED** [review]

File: bitnami/airflow/README.md [hunk 1]
```diff
@@ -976,6 +976,41 @@ The Bitnami Airflow chart relies on the PostgreSQL chart persistence. This means
 | `worker.networkPolicy.ingressNSMatchLabels`                | Labels to match to allow traffic from other namespaces                                                                                                                                                                          | `{}`             |
 | `worker.networkPolicy.ingressNSPodMatchLabels`             | Pod labels to match to allow traffic from other namespaces                                                                                                                                                                      | `{}`             |
 
+### Airflow "setup-db" K8s Job parameters
```
Comment (javsalgar) [inline bitnami/airflow/README.md:979]: I'm missing a documentation section explaining the two options

Comment (juan131) [reply bitnami/airflow/README.md:985]: Done at https://github.com/bitnami/charts/pull/30440/commits/f15f2ec29d0f2ef072e0257a61af260754cc7406

File: bitnami/airflow/README.md [hunk 2]
```diff
@@ -216,6 +216,12 @@ externalDatabase.existingSecret=db-secret
 externalDatabase.existingSecretSqlConnectionKey=sql-connection
 ```
 
+### Database setup
+
+By default, this chart setups the database (init or migrate the schema) and creates the admin user using a K8s job that is created when the chart release is installed or upgraded, and deleted once it success. This job uses [Chart hooks](https://helm.sh/docs/topics/charts_hooks), so it won't be deleted if you're using Helm exclusively for its rendering capabilities (e.g. when using ArgoCD or FluxCD).
```
Comment (javsalgar) [inline bitnami/airflow/README.md:221]: Once it succeeds

Comment (juan131) [reply bitnami/airflow/README.md]: done at https://github.com/bitnami/charts/pull/30440/commits/0d55c99d195fdf3277bb6efc87702264769e9a9e

**Review by javsalgar — APPROVED** [review]

**Pull request merged**